setup.py

```diff
@@ -53,22 +53,7 @@ def extra(self):
         # Uncomment the compiler_directives line below
         # Run: python setup.py build_ext --force --inplace --define CYTHON_TRACE
         ext_modules = cythonize(
-            [
-                'pydantic/class_validators.py',
-                'pydantic/dataclasses.py',
-                'pydantic/datetime_parse.py',
-                'pydantic/env_settings.py',
-                'pydantic/error_wrappers.py',
-                'pydantic/errors.py',
-                'pydantic/fields.py',
-                'pydantic/json.py',
-                'pydantic/main.py',
-                'pydantic/parse.py',
-                'pydantic/schema.py',
-                'pydantic/types.py',
-                'pydantic/utils.py',
-                'pydantic/validators.py',
-            ],
+            ['pydantic/*.py'],
```

Comment:
@samuelcolvin Took me a little while to come up with an approach that was compatible with cython for `pydantic/color.py`, but now every last file is cythonizable!